src/Cli/dotnet/commands/dotnet-workload/install/NetSdkManagedInstaller.cs

```diff
@@ -356,6 +363,11 @@ public IEnumerable<(string, string)> GetInstalledPacks(SdkFeatureBand sdkFeature
                 .Select(packVersionPath => (Path.GetFileName(Path.GetDirectoryName(packVersionPath)), Path.GetFileName(packVersionPath)));
         }
 
+        public void Shutdown()
+        {
+            
```

Comment:
Can we include a comment here that there's no shutdown needed for this install type?

Reply:
Yes